src/components/Post/ClaimPayout.tsx

```diff
@@ -0,0 +1,186 @@
+
+import { LoadingOutlined } from '@ant-design/icons';
+import { web3Accounts, web3Enable, web3FromSource } from '@polkadot/extension-dapp';
+import { InjectedAccountWithMeta } from '@polkadot/extension-inject/types';
+import { Modal, Spin } from 'antd';
+import React, { useContext, useEffect, useState } from 'react';
+import { ApiContext } from 'src/context/ApiContext';
+import { APPNAME } from 'src/global/appName';
+import { NotificationStatus } from 'src/types';
+import AccountSelectionForm from 'src/ui-components/AccountSelectionForm';
+import queueNotification from 'src/ui-components/QueueNotification';
+import Alert from '~src/basic-components/Alert';
+import CustomButton from '~src/basic-components/buttons/CustomButton';
+import { useNetworkSelector } from '~src/redux/selectors';
+import executeTx from '~src/util/executeTx';
+
+interface Props {
+	className?: string;
+	parentBountyId: number | undefined;
+	childBountyId: number | undefined;
+}
+
+const ClaimPayout = ({ className, parentBountyId, childBountyId }: Props) => {
+	const { api, apiReady } = useContext(ApiContext);
+
+	const [showModal, setShowModal] = useState<boolean>(false);
+	const [isLoading, setIsLoading] = useState<boolean>(false);
+	const [availableAccounts, setAvailableAccounts] = useState<InjectedAccountWithMeta[]>([]);
+	const [extensionNotAvailable, setExtensionNotAvailable] = useState<boolean>(false);
+	const [selectedAddress, setSelectedAddress] = useState<string>('');
+	const { network } = useNetworkSelector();
+
+	const getAccounts = async () => {
+		setIsLoading(true);
+		const extensions = await web3Enable(APPNAME);
+
+		if (extensions.length === 0) {
+			setExtensionNotAvailable(true);
+			setIsLoading(false);
+			return;
+		} else {
+			setExtensionNotAvailable(false);
+		}
+
+		const allAccounts = await web3Accounts();
+		setAvailableAccounts(allAccounts);
+		setIsLoading(false);
+	};
+
+	useEffect(() => {
+		getAccounts();
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, []);
+
+	const onAccountChange = (address: string) => {
+		setSelectedAddress(address);
+	};
+
+	const onSuccess = () => {
+		queueNotification({
+			header: 'Success!',
+			message: 'Claim Payout successful.',
+			status: NotificationStatus.SUCCESS
+		});
+		setIsLoading(false);
+		setShowModal(false);
+	};
+
+	const onFailed = (message: string) => {
+		setIsLoading(false);
+		setShowModal(false);
+		queueNotification({
+			header: 'Payout Claim Failed!',
+			message,
+			status: NotificationStatus.ERROR
+		});
+	};
+
+	const handleSignAndSubmit = async () => {
+		if (!selectedAddress || !parentBountyId || !childBountyId || isLoading) return;
+
+		if (!api) {
+			return;
+		}
+
+		if (!apiReady) {
+			return;
+		}
+
+		const injected = await web3FromSource(availableAccounts[0].meta.source);
+
+		api.setSigner(injected.signer);
+
+		setIsLoading(true);
+
+		try {
+			const claim = api.tx.childBounties.claimChildBounty(parentBountyId, childBountyId);
```

Comment:
this tx is incorrect, the tx should be `tx.treasury.payout(payoutIndex)` as discussed